packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_def.hpp

```diff
@@ -396,6 +396,10 @@ namespace MueLu {
       GO numDropped = 0, numTotal = 0;
       std::string graphType = "unamalgamated"; //for description purposes only
 
+      TEUCHOS_TEST_FOR_EXCEPTION(A->GetFixedBlockSize() % A->GetStorageBlockSize() != 0,Exceptions::RuntimeError,"A->GetFixedBlockSize() needs to be a multiple of A->GetStorageBlockSize()");
+      const LO BlockSize = A->GetFixedBlockSize() / A->GetStorageBlockSize();
```

Comment:
Add a comment like so?
"BlockSize is the number of storage blocks that must be kept together during amalgamation."

Reply:
Comment added